README.md

```diff
@@ -10,4 +10,11 @@ sudo apt install libhttp-parser-dev libjansson-dev protobuf-c-compiler \
                  libprotobuf-c-dev
 ```
 
+or on Fedora
+
+```
+sudo dnf install jansson-devel http-parser-devel protobuf-c-devel \
+		protobuf-c-compiler
+```
+
```

Comment:
Nice! I haven't tested `HUNTER_DISABLED=ON` in a while, so I should double-check this still works.